nuxeo-drive-client/nxdrive/manager.py

```diff
@@ -232,13 +232,13 @@ def get_proxies_automatic(self, url):
                 for given URL.
         """
 
-        default = {}
+        default = None
```

Comment:
The old code was returning {} and this is interpreted as no proxy (direct connection) and so even after PAC url is available drive client did not retrieve PAC url. 
So now the get_proxies_automatic method returns 'None'. When PAC url is not found, then drive client will make future attempts at resolving the PAC url. As soon as PAC url is available drive client will automatically go online.